### DISCUSSION TITLE
Εργαστήριο Παρασκευής 04-03-2022

### DISCUSSION THREAD

**Discussion by diogenisAl** [courses-ionio/sw#1045] [Announcements]
Καλησπέρα σας!

Σας υπενθυμίζω για αυτό το εξάμηνο ότι λόγω του μεγάλου πλήθους των φοιτητών/-ιών και του περιορισμένου χώρου, για λόγους ασφάλειας, το δίωρο εργαστήριο της Παρασκευής (17:00-19:00) χωρίζεται σε δύο ίδια μονόωρα:

* την πρώτη ώρα παρακαλούνται να προσέλθουν μόνο όσοι/-ες έχουν Αριθμό Μητρώου που λήγει σε μονό αριθμό (1, 3, 5, 7, 9),
* τη δεύτερη ώρα παρακαλούνται να προσέλθουν μόνο όσοι/-ες έχουν Αριθμό Μητρώου που λήγει σε ζυγό αριθμό (0, 2, 4, 6, 8).

Σε περίπτωση μελλοντικής αλλαγής της συγκεκριμένης οργάνωσης, θα ενημερωθείτε με νέα ανακοίνωση.

**Comment by StavroulaAnatolaki** [comment]
Για οσους ηρθαν ηδη στην ατομικη συναντηση της Τριτης χρειαζεται να ερθουν και σημερα;

**Reply by epidrome** [reply]
το περιεχόμενο των εργαστηρίων και φροντιστηρίων είναι διαφορετικό από τις συναντήσεις του μαθήματος. τα πρώτα έχουν έμφαση στις πρακτικές γνώσεις και δεξιότητες των ασκήσεων, ενώ οι συναντήσεις έχουν έμφαση στην θεωρία και στον τρόπο σκέψης του μαθήματος.
